release/release_gold.sh

```diff
@@ -93,6 +94,12 @@ function main {
 
 	lc_time_run test_boms
 
+	lc_background_run clone_repository liferay-portal-ee
```

Comment:
@lucasmiranda0, be aware of it since it'll affect your current task associated to the GCP migration :wink: